device_doctor/lib/src/android_device.dart

```diff
@@ -16,6 +16,9 @@ import 'host_utils.dart';
 import 'mac.dart';
 import 'utils.dart';
 
+const int _kBatteryMinLevel = 15;
+const int _kBatteryMaxTemperature = 34;
```

Comment:
add units to this name, so that its clear why we are doing the math to work with the 3 digit  android return value.

Reply:
Added, together with some docs to make it clear.